diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index c79181a..7c96f70 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -1882,6 +1882,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
               animatedPosition={animatedPosition}
               backdropComponent={backdropComponent}
             />
+
             <BottomSheetContainer
               key="BottomSheetContainer"
               shouldCalculateHeight={!$modal}
@@ -1892,6 +1893,10 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
               detached={detached}
               style={_providedContainerStyle}
             >
+                           <Animated.View style={[containerStyle,{alignItems: 'center', height: 50, width: '100%',}, ]} >
+                            {props.topItem}
+                            </Animated.View>
+
               <Animated.View style={containerStyle}>
                 <BottomSheetBackgroundContainer
                   key="BottomSheetBackgroundContainer"
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts
index 254f015..ec23f51 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts
@@ -35,6 +35,7 @@ export interface BottomSheetProps
    * @default 0
    */
   index?: number;
+  topItem: React.FC<any>
   /**
    * Points for the bottom sheet to snap to. It accepts array of number, string or mix.
    * String values should be a percentage.
@@ -264,6 +265,8 @@ export interface BottomSheetProps
   onAnimate?: (fromIndex: number, toIndex: number) => void;
   //#endregion
 
+  
+  HeaderChildren?: React.FC<BottomSheetBackdropProps>|null
   //#region components
   /**
    * Component to be placed as a sheet handle.
